mllib/src/main/scala/org/apache/spark/ml/param/shared/SharedParamsCodeGen.scala

```diff
@@ -53,6 +53,7 @@ private[shared] object SharedParamsCodeGen {
       ParamDesc[Int]("checkpointInterval", "checkpoint interval (>= 1)",
         isValid = "ParamValidators.gtEq(1)"),
       ParamDesc[Boolean]("fitIntercept", "whether to fit an intercept term", Some("true")),
+      ParamDesc[Boolean]("standardization", "If the model should perform standardization", Some("true")),
```

Comment:
Technically, It's not performing standardization on the model, maybe we can have more descriptive sentence like the following for this. 
```scala
      ParamDesc[Boolean]("standardization", "whether to standardize the training features" +
        " prior to fitting the model sequence. Note that the coefficients of models are" +
        " always returned on the original scale.", Some("true")),
```